### PR TITLE
Fix default formatter output for critical results

### PR DESCRIPTION
**Pull request by liamg**

### CODE REVIEW

File: internal/app/tfsec/formatters/default.go [hunk 1]
```diff
@@ -19,10 +19,11 @@ import (
...
 
 var severityFormat = map[severity.Severity]string{
-	severity.Low:    tml.Sprintf("<white>%s</white>", severity.Low),
-	severity.Medium: tml.Sprintf("<yellow>%s</yellow>", severity.Medium),
-	severity.High:   tml.Sprintf("<red>%s</red>", severity.High),
-	"":              tml.Sprintf("<white>%s</white>", severity.Low),
+	severity.Low:      tml.Sprintf("<white>%s</white>", severity.Low),
+	severity.Medium:   tml.Sprintf("<yellow>%s</yellow>", severity.Medium),
+	severity.High:     tml.Sprintf("<red>%s</red>", severity.High),
+	severity.Critical: tml.Sprintf("<red>%s</red>", severity.Critical),
```
Comment (owenrumney) [inline internal/app/tfsec/formatters/default.go:25]: Is `BrightRed` an option?

Comment (liamg) [reply internal/app/tfsec/formatters/default.go:25]: There is `lightred`, so i'll flip them round :+1:

Comment (owenrumney) [reply internal/app/tfsec/formatters/default.go:25]: Nice

Comment (liamg) [reply internal/app/tfsec/formatters/default.go]: Used bold instead, worked better

**Review by owenrumney — APPROVED** [review]

**Pull request merged**